File: Programming-languages-and-laboratory/Python/Exercises/43_compito_java_20230221/Visita.py
class Visita:
    def __init__(self, codice_operatore, n_sedute):
        self._codice_operatore = codice_operatore
        self._n_sedute = n_sedute

    def __str__(self):
        return "(" + str(self._codice_operatore) + ", " + str(self._n_sedute) + ")"

File: II - Linguaggi di programmazione e laboratorio/Esame/Java/43_compito_java_20230221/Visita.java
public class Visita {
    protected int codice_operatore;
    protected int n_sedute;

    public Visita(int codice_operatore, int n_sedute) {
        this.codice_operatore = codice_operatore;
        this.n_sedute = n_sedute;
    }

    public String toString() {
        return "(" + codice_operatore + ", " + n_sedute + ")";
    }
}
